ci-operator/step-registry/openshift/edge-tooling/lvms-ci/doctor/openshift-edge-tooling-lvms-ci-doctor-commands.sh

```
@@ -156,6 +156,18 @@ SRC_DIR="${EDGE_TOOLING_DIR}"
 PLUGIN_DIR="${SRC_DIR}/plugins/lvms-ci"
 cd "${SRC_DIR}"
 
+# Run deterministic data collection before Claude.
+# The prepare script collects failed jobs and downloads artifacts.
+# Its JSON summary is saved for the --prepared skill path.
+echo "Running doctor.sh prepare..."
+bash "${PLUGIN_DIR}/scripts/doctor.sh" prepare \
+    --component lvm-operator \
+    --workdir "${WORKDIR}" \
+    ${RELEASE_VERSIONS} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)